### PR TITLE
Moving infra to F39

### PR DESCRIPTION
**Pull request by praiskup**

### CODE REVIEW

**Review by xsuchy — APPROVED** [review]
+1

File: group_vars/dev.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
+---
+copr_instance: dev
+copr_instance_tag: devel
+copr_zone: us-east-1c
+
+backend:
+  name: copr-backend
+  old_instance_id: i-0ba958e718cb91fa3
+  new_instance_id: i-0007e54a85c0e56b1
```
Comment (xsuchy) [inline group_vars/dev.yml:9]: 0007e54a85c0e56b1 is id of current copr-backend-dev. And  0ba958e718cb91fa3 does not exists. Did you already migrated the stage?

File: group_vars/dev.yml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  image: f39
+  temp_volume: vol-03ea76b80451f0e07
```
Comment (xsuchy) [inline group_vars/dev.yml:11]: I would love to see here:
```
# this is volume `data-copr-upgrade-time-dev` that is used as temporary data volume until the newly installed backend is ready to consume real data volume

File: group_vars/dev.yml [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    sdf: vol-01d7d16e5e447b3c4
+    sdg: vol-0ea433a88aef9c644
+    sdh: vol-0038e042c49987b82
+    sdi: vol-0ce8220e998e2e32a
+
+frontend:
+  name: copr-frontend
+  image: f39
+  old_instance_id: i-0327627d968a3bf0d
+  new_instance_id: i-0e737522fecb1ae89
```
Comment (xsuchy) [inline group_vars/dev.yml:33]: Instance summary for i-04b0ccb2b5985bba7 (copr-frontend-prod)

File: group_vars/prod.yml [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  ipv6: 2600:1f18:8ee:ae00:41cc:53dc:b87b:2d8c
+
+  stop_services:
+    before_backup:
+      - copr-dist-git
+      - httpd
+      - crond
+
+  data_volumes:
+    - id: todo
```
Comment (xsuchy) [inline group_vars/prod.yml:76]: This should be 09dd80709532958a8

**Comment by xsuchy** [comment]
I checked the ids, and I there were 1 issue, 1 question, and 1 wish (see above). Beside that, it looks good.

File: group_vars/dev.yml (hunk 1 above)
Comment (praiskup) [reply group_vars/dev.yml:9]: Yes, I already migrated it...

**Comment by praiskup** [comment]
Ok, I think this is ready for merge.  Does anyone want to review (or hit the merge button)?

**Pull request merged**